packages/fx-core/src/plugins/resource/apim/constants.ts

```diff
@@ -280,6 +300,7 @@ export const ProgressMessages: { [key in ProgressStep]: { [step: string]: string
     CreateApim: "Create API Management service",
     CreateAad: "Create client AAD app registration",
   },
+  [ProgressStep.GenerateArmTemplates]: {},
```

Comment:
These message is for progress step. Since we will not add progress step when generating arm templates, will not add message for this section. @XiaofuHuang to review the usage.

Reply:
Fixed.